packages/mdc-tabs/mdc-tabs.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "./tab/mdc-tab";
+@import "./tabs/mdc-tabs";
+@import "./tabs-scroller/mdc-tabs-scroller";
```

Comment:
I'm fine with that. @traviskaufman? Any strong feelings one way or the other? I think it makes sense. If we keep the deviation, it should at least be accounted for in the README.